packages/unity-interface/BrowserInterface.ts

```diff
@@ -409,6 +409,14 @@ export class BrowserInterface {
     getFriendRequests(getFriendRequestsPayload)
   }
 
+  public async MarkMessagesAsSeen(userId: string) {
+    await markAsSeenPrivateChatMessages(userId)
+  }
```

Comment:
nobody catches the failures of this async code.

please change it for:
```suggestion
  public MarkMessagesAsSeen(userId: string) {
    markAsSeenPrivateChatMessages(userId).catch(properHandler)
  }
```